content/developer/reference/backend/orm/changelog.rst

```diff
@@ -13,10 +19,33 @@ Odoo Online version 15.4
   and are less confusing about what is actually done in each case.
   See the section :ref:`SQL Execution <reference/orm/sql>`.
 
+Odoo Online version 15.3
+========================
+
+- Arguments name change for :meth:`~odoo.models.BaseModel.search`, :meth:`~odoo.models.BaseModel.search_count`
+  and :meth:`~odoo.models.BaseModel._search`:  `args` is called `domain` now. (`#83687 <https://github.com/odoo/odoo/pull/83687>`_)
+- :meth:`~odoo.models.BaseModel.filtered_domain` now conserve order of the current recordset (`#83687 <https://github.com/odoo/odoo/pull/83687>`_)
```

Comment:
Let's not add "now" every time because it'll always fit.
```suggestion
- :meth:`~odoo.models.BaseModel.filtered_domain` conserves the order of the current recordset. `#83687 <https://github.com/odoo/odoo/pull/83687>`_
```